profile/README.md

```diff
@@ -0,0 +1,90 @@
+# Rust Bitcoin Community
+
+A Series of projects to implement various Bitcoin protocols in Rust.
+
+## Core Crates
+
+The "core" crates developed and maintained collectively by the `rust-bitcoin` devs.
```

Comment:
Same as below with the "maintainers" argument.
Where do we find about the `rust-bitcoin devs`?

Either link to something or list them?

Reply:
I'm not personally super comfortable defining this group, I think the git index has all the info needed.

Reply:
Then, we can specify somehow that these "devs" are the ones that appear in the commit graph?

Reply:
I've added
```
The "core" crates developed and maintained collectively by the `rust-bitcoin` devs. There is a list
of official [maintainers](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/CONTRIBUTING.md#repository-maintainers)
for the `bitcoin` crate, for other crates please see the git index to get an idea of who is who.
```